app/src/main/java/app/musikus/ui/activesession/ActiveSession.kt

```diff
@@ -11,104 +11,188 @@
 import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.datasource.LoremIpsum
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.LayoutDirection
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import androidx.compose.ui.window.Dialog
-import androidx.compose.ui.window.DialogProperties
 import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import app.musikus.database.LibraryFolderWithItems
+import app.musikus.database.UUIDConverter
 import app.musikus.database.daos.LibraryFolder
+import app.musikus.database.daos.LibraryItem
+import app.musikus.datastore.ColorSchemeSelections
 import app.musikus.ui.Screen
-import app.musikus.ui.activesession.metronome.MetronomeCardBody
-import app.musikus.ui.activesession.metronome.MetronomeCardHeader
-import app.musikus.ui.activesession.recorder.RecorderCardBody
-import app.musikus.ui.activesession.recorder.RecorderCardHeader
+import app.musikus.ui.activesession.metronome.MetronomeUi
+import app.musikus.ui.activesession.recorder.RecorderUi
 import app.musikus.ui.components.DeleteConfirmationBottomSheet
 import app.musikus.ui.components.DialogActions
 import app.musikus.ui.components.DialogHeader
 import app.musikus.ui.components.SwipeToDeleteContainer
+import app.musikus.ui.components.conditional
 import app.musikus.ui.components.fadingEdge
-import app.musikus.ui.library.LibraryItemDialog
 import app.musikus.ui.library.LibraryUiItem
 import app.musikus.ui.sessions.RatingBar
+import app.musikus.ui.theme.MusikusColorSchemeProvider
+import app.musikus.ui.theme.MusikusPreviewElement1
+import app.musikus.ui.theme.MusikusPreviewElement2
+import app.musikus.ui.theme.MusikusPreviewElement3
+import app.musikus.ui.theme.MusikusPreviewElement4
+import app.musikus.ui.theme.MusikusPreviewElement5
+import app.musikus.ui.theme.MusikusPreviewElement6
+import app.musikus.ui.theme.MusikusPreviewWholeScreen
+import app.musikus.ui.theme.MusikusThemedPreview
 import app.musikus.ui.theme.dimensions
+import app.musikus.ui.theme.libraryItemColors
 import app.musikus.ui.theme.spacing
 import app.musikus.utils.DurationFormat
+import app.musikus.utils.TimeProvider
 import app.musikus.utils.UiIcon
 import app.musikus.utils.UiText
 import app.musikus.utils.getDurationString
-import kotlin.time.Duration
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.ImmutableMap
+import kotlinx.collections.immutable.persistentListOf
+import kotlinx.collections.immutable.toImmutableList
+import kotlinx.collections.immutable.toImmutableMap
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.launch
+import java.time.ZonedDateTime
+import java.util.UUID
+import kotlin.random.Random
+import kotlin.time.Duration.Companion.seconds
```

Comment:
Holy schmoly 🤯

Reply:
?

`UUID` is needed in L#1248
`ZonedDateTime` is needed in L#1432 and L#1696
`Random` is needed in various previews to generate random content
`seconds` is needed in `PreviewActiveSessionScreen`

I have auto-import activated, so all imports should actually be needed somewhere

Reply:
It's not a critique 😅 just a comment